src/main/minio.js

```diff
@@ -470,6 +487,14 @@ export class Client {
 
     var payload = ''
 
+    // Region already set in constructor, validate if
+    // caller requested bucket location is same.
+    if (this.region) {
+      if (region && region !== this.region) {
```

Comment:
Changing `makeBucket()` behavior can happen later.  We need to allow for variadic arguments for that.